packages/cap-config-writer/src/cap-config/package-json.ts

```diff
@@ -61,7 +61,11 @@ export function addCdsPluginUi5(packageJson: Package): void {
  * @returns - true: min cds version is present; false: cds version needs update
  */
 export function hasMinCdsVersion(packageJson: Package): boolean {
-    return gte(coerce(packageJson.dependencies?.['@sap/cds']) ?? '0.0.0', minCdsVersion);
+    const packageCdsVersion = packageJson.dependencies?.['@sap/cds'];
+    return (
+        satisfies(minCdsVersion, packageCdsVersion ?? '0.0.0') ||
```

Comment:
`satisifies` allows all versions within a range, we need a min version for the check.